### PR TITLE
Add Code from Old branch to new Package branch

### PR DESCRIPTION
**Pull request by jvanbaarsen**
* Better code displaying
* Grid, Markdown and message updates
* Box, button, grid, message, pagebar and config updates
* Code highligthing, markdown, special gradient updates

### CODE REVIEW

File: src/components/background.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  background-image:
+    linear-gradient(
+      to right,
+      red,
+      red 33%,
+      green 33%,
+      green 85%,
+      orange 85%
+    );
+}
```
Comment (wesoudshoorn) [inline src/components/background.css]: I think this whole document is not used, let's look up these strings

File: src/components/grid.css [hunk 2]
```diff
@@ -1,38 +1,35 @@
-.grid-app {
+/*  Grid for docs */
```
Comment (wesoudshoorn) [inline src/components/grid.css]: The whole grids will probably need some refactorign.

File: src/design-system.css [hunk 3]
```diff
@@ -5,14 +5,17 @@
 @import "./defaults/fonts";
 
 @import "./components/avatar";
+@import "./components/background";
```
Comment (wesoudshoorn) [inline src/design-system.css]: Coupled to first comment

File: tailwind.config.js [hunk 4]
```diff
@@ -24,6 +24,18 @@ module.exports = {
...
+      "md": " 0 4px 6px -1px rgba(10,21,48, .1), 0 2px 4px -1px rgba(10,21,48, .06)",
+      "lg": " 0 10px 15px -3px rgba(10,21,48, .1), 0 4px 6px -2px rgba(10,21,48, .05)",
+      "xl": " 0 20px 25px -5px rgba(10,21,48, .1), 0 10px 10px -5px rgba(10,21,48, .04)",
+      "2xl": "0 25px 50px -12px rgba(10,21,48, .25)",
+      "3xl": "0 35px 60px -15px rgba(10,21,48, .3)",
+      "inner": "inset 0 2px 4px 0 rgba(10,21,48,0.06)",
+      "outline": "0 0 0 3px rgba(66,153,225,0.5)",
+      "focus": "0 0 0 3px rgba(66,153,225,0.5)",
+      "none": "none"
+    },
```
Comment (wesoudshoorn) [inline tailwind.config.js]: For context: this changes all the black shadows to dark blue shadows.

File: src/components/code-highlight.css [hunk 5]
```diff
@@ -0,0 +1,103 @@
+/*
+// Solarized Light
```
[the remark this answers is not in the document]
Comment (jvanbaarsen) [reply src/components/code-highlight.css]: Issue is here: https://github.com/appsignal/design-system/issues/6

**Pull request merged**